### PR TITLE
23.0: docs/state-sync: Update state sync docs

### PR DESCRIPTION
**Pull request by ptrus**
State sync config is simplified in v23.x, specifying node addresses is not needed anymore.

Note: due to an unrelated libp2p issue (to be fixed) this currently doesn't work on testnet.

---

Wait with merging until after Eden upgrade.

### CODE REVIEW

**Review by kostko — APPROVED** [review]
Looks good to me, make sure to only merge after the Eden upgrade is live.

**Comment by matevz** [comment]
FYI you will probably have to rebase your branch because https://github.com/oasisprotocol/docs/pull/588 will get in before the 23.0 upgrade on Mainnet.

File: docs/node/run-your-node/advanced/sync-node-using-state-sync.md [hunk 1]
```diff
@@ -128,24 +106,3 @@ Query our public Rosetta Gateway instance and obtain the trusted height and hash
 Query our public Oasis Node's gRPC endpoint and obtain the trusted height and hash there:
 
 1. _TODO._
-
```
Comment (matevz) [inline docs/node/run-your-node/advanced/sync-node-using-state-sync.md:162]: I know this probably isn't your markdown, but what's with the TODOs above? The public gRPCs we use for our wallets won't work?

**Review by matevz — APPROVED** [review]

File: docs/node/run-your-node/advanced/sync-node-using-state-sync.md (hunk 1 above)
Comment (ptrus) [reply docs/node/run-your-node/advanced/sync-node-using-state-sync.md:162]: Probably a TODO for the example (e.g. show how to query the gRPC endpoint to obtain the height and hash). I can add it.

Comment (ptrus) [reply docs/node/run-your-node/advanced/sync-node-using-state-sync.md:162]: Updated

File: docs/node/run-your-node/advanced/sync-node-using-state-sync.md [hunk 2]
```diff
@@ -151,43 +126,59 @@ The values you need are `latest_height` and `latest_hash` .
...
 
 #### Oasis Node's gRPC Endpoint
 
 Query our public Oasis node's gRPC endpoint and obtain the trusted height and
 hash there:
 
-1. _TODO._
 
-### Obtaining Addresses of Oasis Nodes' Publicly Exposed gRPC Endpoints
+```go
```
Comment (kostko) [inline docs/node/run-your-node/advanced/sync-node-using-state-sync.md]: One could also just use the Oasis CLI for this? E.g. running `oasis network status | jq '.consensus.latest_height, .consensus.latest_hash'` gets you:
```
18466380
"0589164bc3c13fc919c2f0d7fd2b850161366cbabfa0a22d54fda215a38167c1"
```

Comment (ptrus) [reply docs/node/run-your-node/advanced/sync-node-using-state-sync.md]: updated

File: docs/node/run-your-node/advanced/sync-node-using-state-sync.md [hunk 3]
```diff
@@ -151,43 +126,59 @@ The values you need are `latest_height` and `latest_hash` .
...
-
-If you already have a local Oasis node set up, you can list the descriptors of
-all registered nodes via the [`oasis network show nodes`] Oasis CLI command.
+```bash
+curl -X POST https://rosetta.oasis.dev/api/block \
+-H "Content-Type: application/json" \
+-d '{
+    "network_identifier": {
+        "blockchain": "Oasis",
+        "network": "b11b369e0da5bb230b220127f5e7b242d385ef8c6f54906243f30af63c815535"
```
Comment (kostko) [inline docs/node/run-your-node/advanced/sync-node-using-state-sync.md]: Maybe mention that this is the chain context and where to look it up (links for mainnet and testnet). Also we should update the example after the Eden upgrade to include the new chain context.

**Review by kostko — APPROVED** [review]

File: docs/node/run-your-node/advanced/sync-node-using-state-sync.md (hunk 3 above)
Comment (ptrus) [reply docs/node/run-your-node/advanced/sync-node-using-state-sync.md]: Added

**Comment by kostko** [comment]
We can merge this now since we're also updating the configs.

**Pull request merged**